rtic_v5/hid_mouse_stm32f0/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "stm32f042"
+version = "0.1.0"
+authors = ["Roman Valls Guimera <brainstorm@nopcode.org>", Josh Johnson]
+edition = "2018"
+
+[dependencies.defmt]
+git = "https://github.com/knurling-rs/defmt"
+branch = "main"
+
+[dependencies]
+embedded-hal = "0.2.4"
+usb-device = "0.2.5"
+usbd-hid = "0.4.4"
+panic-semihosting = "0.5"
```

Comment:
Is `panic-semihosting` and `cortex-m-semihosting` still required? Seems you are using panic-halt and `rtt-target` instead.